src/components/FeatureViews/Accounts/TabComponents/InstructorCourses.js

```diff
@@ -15,6 +15,14 @@ import Moment from "react-moment";
 
 import {courseDateFormat} from "utils";
 
+export const GET_INSTRUCTOR_ENROLLMENTS = `
+  query InstructorEnrollments($instructorId) { 
+    courses(filter: { instructor.user.id: $instructorId){
```

Comment:
Is this formatted properly?

Reply:
this is still a work in progress haha. not done with this one, but also not sure what the syntax should be here